runhouse/resources/hardware/cluster.py

```diff
@@ -216,13 +216,22 @@ def save_config_to_cluster(
 
         json_config = f"{json.dumps(config)}"
 
-        self.run(
+        save_config_outputs: list = self.run(
             [
                 f"mkdir -p ~/.rh; touch {CLUSTER_CONFIG_PATH}; echo '{json_config}' > {CLUSTER_CONFIG_PATH}"
             ],
             node=node or "all",
         )
 
+        for output in save_config_outputs:
+            for res in output:
+                # if run_res[1] or run_res[2] are not empty strings, that means we got an error while saving the
+                # cluster config
+                if res[1] or res[2]:
+                    logger.error("Failed to save cluster config locally")
+                    logger.error(res[1] + res[2])
+                    raise ValueError()
```

Comment:
Kinda on the fence here about raising an exception. On the one hand, if some error occurs while saving the config, we want to kill the cluster setup, cause the ClustetServlet setup depends on the local config saved on the cluster. One the other hand, this check is for internal Runhouse usage and debuggability, not sure we need to surface this error to the user by raising an exception.

Reply:
i would lean towards the latter and not raising an exception in this case as you noted, and keeping the behavior for now as is